refine_dictionaries/get_closest_words.py

```diff
@@ -0,0 +1,373 @@
+       - dem_concat (pandas df)= demographic dataframe with the columns 'term', 'cosine_sim_with_core', ' closest_terms_from_core'
+       - relt_concat (pandas df) = relational dataframe with the columns 'term', 'cosine_sim_with_core', ' closest_terms_from_core'
+    
+    """
+    # remove terms from each dictionary not in w2v model vocab
+    
+    dem_core=remove_terms(dem_core, model)
+    cult_core=remove_terms(cult_core, model)
+    relt_core=remove_terms(relt_core, model)
+    
+    if get_500:
+        cult_dict = [i[0] for i in model.wv.most_similar(cult_core, topn = 500)]
+        dem_dict = [i[0] for i in model.wv.most_similar(dem_core, topn = 500)]
+        relt_dict = [i[0] for i in model.wv.most_similar(relt_core, topn = 500)]
+    
+    cult_concat=get_output_df(cult_dict, cult_core,model)
+    dem_concat=get_output_df(dem_dict, dem_core,model)
+    relt_concat=get_output_df(relt_dict, relt_core,model)
+    return cult_concat, dem_concat, relt_concat
+
+
+
+####################################
+###### get similarities of expanded dicts (500)
+####################################
+
+def save_expanded_dicts_df():
+    '''
+    save csv's for the similarity metrics of expanded (500) terms
+    '''
+    # run similarities function over all three dicts for each decade
+    
+    cult_exp1, dem_exp1, relt_exp1=get_results_decade(m1, cult_core, dem_core, relt_core, [], [], [], get_500=True)
+
+    cult_exp2, dem_exp2, relt_exp2=get_results_decade(m2, cult_core, dem_core, relt_core, [], [], [], get_500=True)
+
+    cult_exp3, dem_exp3, relt_exp3=get_results_decade(m3, cult_core, dem_core, relt_core, [], [], [], get_500=True)
+
+    cult_exp4, dem_exp4, relt_exp4=get_results_decade(m4, cult_core, dem_core, relt_core, [], [], [], get_500=True)
+
+    cult_exp1.to_csv(output_dir+"/cos_sim_df_expanded_500_cult_1971-1981.csv")
+    dem_exp1.to_csv(output_dir+"/cos_sim_df_expanded_500_dem_1971-1981.csv")
+    relt_exp1.to_csv(output_dir+"/cos_sim_df_expanded_500_relt_1971-1981.csv")
+
+    cult_exp2.to_csv(output_dir+"/cos_sim_df_expanded_500_cult_1982_1992.csv")
+    dem_exp2.to_csv(output_dir+"/cos_sim_df_expanded_500_dem_1982_1992.csv")
+    relt_exp2.to_csv(output_dir+"/cos_sim_df_expanded_500_relt_1982_1992.csv")
+
+    cult_exp3.to_csv(output_dir+"/cos_sim_df_expanded_500_cult_1993_2003.csv")
+    dem_exp3.to_csv(output_dir+"/cos_sim_df_expanded_500_dem_1993_2003.csv")
+    relt_exp3.to_csv(output_dir+"/cos_sim_df_expanded_500_relt_1993_2003.csv")
+
+    cult_exp4.to_csv(output_dir+"/cos_sim_df_expanded_500_cult_2004_2014.csv")
+    dem_exp4.to_csv(output_dir+"/cos_sim_df_expanded_500_dem_2004_2014.csv")
+    relt_exp4.to_csv(output_dir+"/cos_sim_df_expanded_500_relt_2004_2014.csv")
+
+
+####################################
+###### get similarities of refined dicts
+####################################
+
+
+def save_refined_dicts_df():
```

Comment:
The function names should make more clear what they are doing (not just saving), something more like below (drop the `df` part for simplicity). 

```suggestion
def get_results_save_refined_dicts():
```